cmd/migration.go

```diff
@@ -37,8 +44,15 @@ var migrateDownCmd = &cobra.Command{
 		logger := log.NewLogger("buroq-migrate")
 		mSource := getMigrateSource()
 
-		doMigrate(appCtx, logger, mSource, appcontext.DBDialectMysql, migrate.Down)
-		doMigrate(appCtx, logger, mSource, appcontext.DBDialectPostgres, migrate.Down)
+		switch {
```

Comment:
1. better also check is_enabled
2. suggest using if statement rather than switch. So if it enable, then run.

Reply:
done